pkg/apitype/updates.go

```diff
@@ -205,6 +205,7 @@ type PatchUpdateCheckpointRequest struct {
 }
 
 // AppendUpdateLogEntryRequest defines the body of a request to the append update log entry endpoint of the service API.
+// No longer sent from, but the type definition is still required for backwards compat with older clients.
```

Comment:
NIT: "No longer sent from" read a little weird.  Did you drop something like "the CLI"

Reply:
Looks like it 😬 . Done.